# Leyes de la termodinámica

* **Ley 0 de la termodinámica:** Si pones dos cuerpos juntos intercambiarán calor hasta llegar a la misma temperatura.

El equilibrio térmico se debe a ley 0.

* **Ley 1 de la termodinámica:** Le energía no se crea ni se destruye, sólo se transforma.

$$ \Delta U=\Delta Q-\Delta W $$

* $\Delta U$: energía interna
* $\Delta Q$: "qualor" 
* $\Delta W$: Work (trabajo)

# Equilibrio térmico

Vamos a seguir a trabajando con la fórmula de específico.

$$ Q=m\cdot C\cdot\Delta T $$

* $Q$ : "qualor"
* $m$ : masa
* $\Delta T$ : diferencia de temperatura
* $C$ : específico

$$ \Delta T=T_f-T_i $$

* $T_i$ : temperatura inicial
* $T_f$ : temperatura final

<img src="recta1.png" align="middle" width="1000">

* **Ejemplo:** Si a una vaso con $0.5kg$ de agua le damos $300J$ de "qualor", ¿cuánto aumenta su temperatura?

* Datos:
    * $m=0.5kg$
    * $Q=300J$
    * $\Delta T=?$
    * $C=4186 J/(kg\cdot K)$

* Fórmula:

$$ Q=m\cdot C\cdot\Delta T $$

$$ \Delta T=\dfrac{Q}{m\cdot C}=0.14K=0.14^\circ C $$

In [1]:
300/(0.5*4186)

0.1433349259436216

* **Ejemplo:** Una sartén de acero de $0.2kg$  está a una temperatura de $200^\circ C$. Le ponemos $0.5kg$ de agua a $10^\circ C$. ¿A qué temperatura llega el sartén?

* Datos:
    * $m_\text{acero}=0.2kg$
    * $m_\text{agua}=0.5kg$
    * ${T_i}_\text{acero}=200^\circ C$
    * ${T_i}_\text{agua}=10^\circ C$
    * ${T_f}_\text{acero}=?$
    * ${T_f}_\text{agua}=?$
    * $Q_\text{acero}=?$ (pero no importa)
    * $Q_\text{agua}=?$ (pero no importa)
    * $C_\text{acero}=480J/(kg\cdot K)$
    * $C_\text{agua}=4186J/(kg\cdot K)$

* Fórmula:

$$ Q_\text{acero}=m_\text{acero}\cdot C_\text{acero}\cdot\Delta T_\text{acero} $$

$$ Q_\text{agua}=m_\text{agua}\cdot C_\text{agua}\cdot\Delta T_\text{agua} $$

$$ \Delta T=T_f-T_i $$

En los problema de equilibrio térmico, se igualan los "qualores", uno con signo positivo y el otro con signo negativo.

$$ Q_\text{acero}=-Q_\text{agua} $$

$$ m_\text{acero}\cdot C_\text{acero}\cdot\Delta T_\text{acero}=-m_\text{agua}\cdot C_\text{agua}\cdot\Delta T_\text{agua} $$

En los problema de equilibrio térmico,la temperatura final de los objetos es 
la misma.

$$ m_\text{acero}\cdot C_\text{acero}\cdot({T_f}-{T_i}_\text{acero})=-m_\text{agua}\cdot C_\text{agua}\cdot({T_f}-{T_i}_\text{agua}) $$

Vamos a despejar temperatura final:

$$ m_\text{acero}\cdot C_\text{acero}\cdot({T_f}-{T_i}_\text{acero})=-m_\text{agua}\cdot C_\text{agua}\cdot({T_f}-{T_i}_\text{agua}) $$

$$ m_\text{acero}\cdot C_\text{acero}\cdot{T_f}-m_\text{acero}\cdot C_\text{acero}\cdot{T_i}_\text{acero}=-m_\text{agua}\cdot C_\text{agua}\cdot{T_f}+m_\text{agua}\cdot C_\text{agua}\cdot{T_i}_\text{agua} $$

$$ m_\text{acero}\cdot C_\text{acero}\cdot{T_f}+m_\text{agua}\cdot C_\text{agua}\cdot{T_f}=m_\text{acero}\cdot C_\text{acero}\cdot{T_i}_\text{acero}
+m_\text{agua}\cdot C_\text{agua}\cdot{T_i}_\text{agua} $$

$$ (m_\text{acero}\cdot C_\text{acero}+_\text{agua}\cdot C_\text{agua})\cdot{T_f}=m_\text{acero}\cdot C_\text{acero}\cdot{T_i}_\text{acero}+m_\text{agua}\cdot C_\text{agua}\cdot{T_i}_\text{agua} $$

$$ {T_f}=\dfrac{m_\text{acero}\cdot C_\text{acero}\cdot{T_i}_\text{acero}+m_\text{agua}\cdot C_\text{agua}\cdot{T_i}_\text{agua}}{m_\text{acero}\cdot C_\text{acero}+m_\text{agua}\cdot C_\text{agua}} $$

* Datos:
    * $m_\text{acero}=0.2kg$
    * $m_\text{agua}=0.5kg$
    * ${T_i}_\text{acero}=200^\circ C$
    * ${T_i}_\text{agua}=10^\circ C$
    * $C_\text{acero}=480J/(kg\cdot K)$
    * $C_\text{agua}=4186J/(kg\cdot K)$

In [5]:
(0.2*480*200+0.5*4186*10)/(0.2*480+0.5*4186)

18.332571950662402

* $T_f=18.33^\circ C$

# Proceso isotérmico

Se trata de tener la misma temperatura.

# Proceso isocórico

Se trata de un tener el mismo volumen.

# Proceso isobárico

Se trata de un tener la misma presión.